ts/private/ts_lib.bzl

```diff
@@ -3,6 +3,58 @@
 load("@aspect_rules_js//js:libs.bzl", "js_lib_helpers")
 load("@aspect_rules_js//js:providers.bzl", "JsInfo")
 
+resource_set_values = [
+    "default",
+    "mem_512m",
+    "mem_1g",
+    "mem_2g",
+    "mem_4g",
+    "mem_8g",
+    "mem_16g",
+    "mem_32g",
+]
+
+def _resource_set_mem_512m(_, __):
+    return { "memory": 512 }
+
+def _resource_set_mem_1g(_, __):
+    return { "memory": 1024 }
+
+def _resource_set_mem_2g(_, __):
+    return { "memory": 2048 }
+
+def _resource_set_mem_4g(_, __):
+    return { "memory": 4096 }
+
+def _resource_set_mem_8g(_, __):
+    return { "memory": 8192 }
+
+def _resource_set_mem_16g(_, __):
+    return { "memory": 16384 }
+
+def _resource_set_mem_32g(_, __):
+    return { "memory": 32768 }
+
+# buildifier: disable=function-docstring
+def resource_set(name):
```

Comment:
if you make this `_resource_set` you don't need the buildifier disable

Reply:
but it is loaded by another file so the symbol can't be private